install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -429,6 +429,7 @@ data:
   routing-mode: "native"
   enable-endpoint-routes: "true"
   enable-local-node-route: "false"
+  enable-health-check-loadbalancer-ip: "true"
```

Comment:
Seems like this section is for routing / tunneling options, see my next comment.